express-api/src/services/projects/projectsServices.ts

```diff
@@ -877,9 +870,26 @@ const getProjectsForExport = async (filter: ProjectFilter, includeRelations: boo
     skip: (filter.page ?? 0) * (filter.quantity ?? 0),
     order: filter.sort as FindOptionsOrder<Project>,
   };
-
   const projects = await projectRepo.find(queryOptions);
-  return projects;
+
+  /**
+   * Separated these queries intentionally.
+   * Joining them with the projects call is much slower.
+   */
+  const tasks = await AppDataSource.getRepository(ProjectTask).find();
+  const notes = await AppDataSource.getRepository(ProjectNote).find();
+  const monetaries = await AppDataSource.getRepository(ProjectMonetary).find();
+  const timestamps = await AppDataSource.getRepository(ProjectTimestamp).find();
+  const notifications = await AppDataSource.getRepository(NotificationQueue).find();
+
+  return projects.map((project) => ({
+    ...project,
+    Tasks: tasks.filter((task) => task.ProjectId === project.Id),
```

Comment:
This works, but it's probably more efficient to filter using `find({ where: { ProjectId: project.Id} })` than doing the js filter.